content/blog/research-community-forum-3-june-2023.md

```diff
@@ -7,14 +7,11 @@ authors:
   - rufuspollock
 image: /assets/images/research-forum-3.jpg
 ---
+>This video forms part of our series of [[/research|research community]] monthly fora.
 
-Jonathan Reams will present at the third [[/research/|research community forum]]. Jonathan is Associate Professor at the Norwegian University of Science and Technology and has a wealth of expertise spanning many areas across adult and leadership development, consciousness research and transformative learning. Jonathan practices the cultivation of leadership through consulting and leadership development program design and delivery. He brings awareness based technology to this work, focusing on how the inner workings of human nature can develop leadership capacities for today’s complex challenges. He is one of the leading experts in Europe on [Lectica](https://lecticalive.org), one of the main frameworks for adult developmental assessment.
```

Comment:
@laurenwigmore we wouldn't remove this info - we'd just add the new info at the bottom of the post in a section called e.g. "Post event info"

Reply:
OK, i now see you've just split things apart, not removed stuff. That makes sense!